Lib/ipaddress.py

```diff
@@ -1519,7 +1519,13 @@ def __init__(self, address, strict=True):
         # Constructing from a tuple (addr, [mask])
         elif isinstance(address, tuple):
             addr = address[0]
-            mask = address[1] if len(address) > 1 else self._max_prefixlen
+            if len(address) > 1:
+                if isinstance(address[1], int) and 0 < address[1] > 32:
+                    raise NetmaskValueError("The netmask should be isn't valid")
```

Comment:
This error message doesn't read well.

Reply:
Yes, this can be reworded and could include the max value (32) in the error so that users know the limit.

Reply:
really strange, that was not my original message. Maybe a bad paste :/